Sources/Responses/EventItemType.swift

```diff
@@ -22,6 +22,8 @@ public class EventItem: BoxModel {
         case file(File)
         /// Comment type
         case comment(Comment)
+        /// User type
```

Comment:
I think coveralls is complaining for code coverage, maybe a test case if event type is deserialized into User object?

Reply:
added test